src/core/public/chrome/recently_accessed/recently_accessed_service.ts

```diff
@@ -56,11 +58,13 @@ export class RecentlyAccessedService {
 
     return {
       /** Adds a new item to the history. */
-      add: (link: string, label: string, id: string) => {
+      add: (link: string, label: string, id: string, type?: string, updatedAt?: string) => {
```

Comment:
```suggestion
      add: (link: string, label: string, id: string, extraProps?: { type?: string; updatedAt?: string }) => {
```
What about we using a object? Or the arguments list may be too long to use.

Reply:
Changed